css/style.css

```diff
@@ -61,19 +59,213 @@ body {
 
 /* Center the container */
 .container {
-  width: 960px;
-  margin: auto;
-  height: 500px;
+  width: 100%;
+  margin: 0 auto;
+  height: 600px;
+  background: url(../images/header_bg.jpg) no-repeat center top rgb(134,141,157);
+  background-size: cover;
+  float:left;
 }
 
 header {
+  height:600px;
+  margin:0px auto;
+  padding-top:50px;
+  padding-left:40px;
+}
+
+.tagline {
+  padding-left:-40px;
+}
+
+
+.button {
+  font-size: 14px;
+  font-weight: 800;
+  text-align: center;
+  border: 7px solid #f9e42e;
+  border-radius: 5px;
+  background-color: #f9e42e;
+  margin: 0% 45%;
+}
+
+.button a:link {
+  color: #121212;
+  text-decoration: none;
+}
+
+.button a:visited {
+  color: #121212;
+  text-decoration: none;
+}
+
+.button a:hover {
+  color: #121212;
+  text-decoration: none;
+}
+
+.section2 {
+  width: 100%;
+  background-size: cover;
+  background-color: #f0efef;
+  margin: auto;
+  height: 350px; 
+  float: left;
+}
 
+.benefits {
+  width: 60%;
+  margin-top: 0px;
+  padding-top: 50px;
+  margin-left: 40px;
+}
+
+h1 {
+  font-size: 24px;
+  font-weight: 800;
+  color: #FFF;
+}
+
+h2 {
+  font-weight: 600;
+  font-size: 24px;
+  color: #FFF;
+  padding-top: 150px;
+  padding-bottom: 30px;
+  text-align: center;
+}
+
+h3 {
+  font-size: 20px;
+  color: #033048;
+  font-weight: 600;
+}
+
+p {
+  font-size: 13px;
+  font-weight: 300;
+  color: #606161;
+}
+
+#subhead {
+  margin-top: 20px;
+}
+
+li {
+  font-size: 13px;
+  color: #606161;
+  font-weight: 300;
+  margin-top: 20px;
+  margin-left: 65px;
+}
+
+.section3 {
+  width: 100%;
+  background: url(../images/testimonial_bg.jpg) no-repeat center;
+  background-size: cover;
+  margin: 0 auto;
+  height: 200px; 
+  display: inline-block;
+
+}
+  
+.testimonial {
+  width: 35%;
+  padding-top: 60px;
+  margin: auto;
+  padding-bottom: 40px;
+}
+
+
+h4 {
+  font-size: 18px;
+  font-weight: 300;
+  font-style: italic;
+  text-align: center;
+  color: #f0efef;
+}
+
+h5 {
+  font-size: 18px;
+  font-weight: 600;
+  text-align: center;
+  color: #f0efef;
+}
+
+.section4 {
+  width: 100%;
```

Comment:
@jonpackles The default with of an element is 100% so it is not needed to define that.